### PR TITLE
show playerCount in code point leaderboard

### PR DESCRIPTION
**Pull request by mrfinch**
Screenshot 2021-02-03 at 1 22 24 PM

### CODE REVIEW

File: app/core/store/modules/seasonalLeague.js [hunk 1]
```diff
@@ -356,6 +365,12 @@ export default {
       await topCodePointsRankingPromise
 
       commit('setCodePointsRanking', { leagueId: leagueId, ranking: codePointsRankingInfo })
+
+      getCodePointsPlayerCount(leagueId)
+          .then((playerCount) => {
+            commit('setCodePointsPlayerCount', Number(playerCount));
```
Comment (AndrewJakubowicz) [inline app/core/store/modules/seasonalLeague.js:371]: Not blocking but curious if there is a difference between `Number` and `parseInt` for conversion of string to number.

**Review by AndrewJakubowicz — APPROVED** [review]
LGTM! Also tested manually with proxy and works great!

File: app/core/store/modules/seasonalLeague.js [hunk 2]
```diff
@@ -163,7 +169,10 @@ export default {
         }
         return []
       }
-    }
+    },
+    codePointsPlayerCount(state) {
+      return state.codePointsRankingsForLeague.playerCount;
```
Comment (nwinter) [inline app/core/store/modules/seasonalLeague.js:174]: Not sure if it's here or in the template, but we should use `playerCount.toLocaleString()` to add the appropriate decimal separators in there. Screenshots just 222265 players, but for readability when numbers get past 10K, we want that comma: 222,265 players.

This should apply to other places we display counts, it's just that there are rarely such large numbers! 🥳

**Review by nwinter — CHANGES_REQUESTED** [review]

File: app/core/store/modules/seasonalLeague.js (hunk 1 above)
Comment (mrfinch) [reply app/core/store/modules/seasonalLeague.js:371]: I just searched for it. There is a difference
```
// parsing:
parseInt("20px");       // 20
parseInt("10100", 2);   // 20
parseInt("2e1");        // 2

// type conversion
Number("20px");       // NaN
Number("2e1");        // 20, exponential notation
```

File: app/core/store/modules/seasonalLeague.js (hunk 2 above)
Comment (mrfinch) [reply app/core/store/modules/seasonalLeague.js:174]: made the change, merging.

**Pull request merged**